src/application/services/useNote.ts

```diff
@@ -265,6 +278,35 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     parentNote.value = undefined;
   }
 
+  /**
+   * Reform the array of note parents by adding the actual note id and content.
```

Comment:
still don't understand the problem. API returns id and content, why do we need to change them?